lib/compliance/flows/id-card-data.js

```diff
@@ -10,34 +10,44 @@ const idDataMachine = xstate.Machine({
   initial: 'idle',
   strict: true,
   states: {
-    idle: {on: {START: 'scanId'}},
+    idle: { on: { START: 'scanId' } },
     scanId: {
       onEntry: ['timeoutToScannerCancel', 'transitionScreen', 'scanPDF'],
       on: {
-        SCANNED: {'authorizing': {actions: ['transitionScreen', 'authorizeIdCardData']}},
-        SCAN_ERROR: {'idScanFailed': {actions: ['timeoutToFail', 'transitionScreen']}},
+        SCANNED: { 'authorizing': { actions: ['transitionScreen', 'authorizeIdCardData'] } },
+        SCAN_ERROR: { 'idScanFailed': { actions: ['timeoutToFail', 'transitionScreen'] } },
         FAIL: 'failure'
       }
     },
     authorizing: {
       on: {
         AUTHORIZED: 'success',
-        BLOCKED_ID: {'idVerificationFailed': {actions: ['timeoutToFail', 'transitionScreen']}}
+        BLOCKED_ID: { 'idVerificationFailed': { actions: ['timeoutToFail', 'transitionScreen'] } }
       }
     },
-    idScanFailed: {on: {FAIL: 'failure'}},
-    idVerificationFailed: {on: {FAIL: 'failure'}},
+    idScanFailed: { on: { FAIL: 'failure' } },
+    idVerificationFailed: { on: { FAIL: 'failure' } },
     failure: { onEntry: ['failure'] },
     success: { onEntry: ['success'] }
   }
 })
 
 let currentStateValue
 let data
+let idCardDataStills = []
 
 function getData () { return data }
 function setData (value) { data = value }
 
+function setIdDataPhotos (photo) {
```

Comment:
Hmm, function name should be `addIdDataPhoto()`, as we're not really setting anything, we're adding to an array